pod.go

```diff
@@ -711,11 +713,7 @@ func (p *Pod) enter(args []string) error {
 	return nil
 }
 
-func (p *Pod) setPodState(state stateString) error {
-	p.state = State{
-		State: state,
-	}
-
+func (p *Pod) setPodState() error {
```

Comment:
Either we keep that method name and we keep its argument or we rename it to `storePodState()`. I'd prefer the former.